ui/components/app/account-menu/account-menu.component.js

```diff
@@ -400,6 +409,33 @@ export default class AccountMenu extends Component {
           text={t('connectHardwareWallet')}
         />
         <div className="account-menu__divider" />
+        {
+          ///: BEGIN:ONLY_INCLUDE_IN(flask)
+          <>
+            <AccountMenuItem
+              onClick={() => {
+                toggleAccountMenu();
+                history.push(NOTIFICATIONS_ROUTE);
+              }}
+              icon={
+                <div className="account-menu__notifications">
+                  <i
+                    className="fa fa-bell fa-xl"
+                    color="var(--color-icon-default)"
```

Comment:
Does this `color` property work on a native HTML element?

Reply:
Yes it works on the font-awesome icons 👍